### ISSUE TITLE
bug: install with pip bug

### ISSUE THREAD

**Issue by antonylebechec** [natir/variantplaner#38]
### Description of the bug
During installation (pip install...), a bug

### To Reproduce
```
pip install git+https://github.com/natir/variantplaner.git@0.2.2#egg=variantplaner
```

### Full traceback
```
Compiling polars-error v0.33.2
error[E0658]: use of unstable library feature 'int_log'
  --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/polars-utils-0.33.2/src/cache.rs:62:27
   |
62 |             shift: 64 - n.ilog2(),
   |                           ^^^^^
   |
  = note: see issue #70887 <https://github.com/rust-lang/rust/issues/70887> for more information
```

### Expected behavior
Installation ok

### Environment information
Python 3.10
Rust 1.66.1

**Comment by antonylebechec** [comment]
Same error with pipx:

```
         Compiling comfy-table v7.0.1
         Compiling polars-error v0.33.2
      error[E0658]: use of unstable library feature 'int_log'
        --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/polars-utils-0.33.2/src/cache.rs:62:27
         |
      62 |             shift: 64 - n.ilog2(),
         |                           ^^^^^
         |
         = note: see issue #70887 <https://github.com/rust-lang/rust/issues/70887> for more information

      For more information about this error, try `rustc --explain E0658`.
      error: could not compile `polars-utils` due to previous error
      warning: build failed, waiting for other jobs to finish...
      💥 maturin failed
        Caused by: Failed to build a native library through cargo
        Caused by: Cargo build finished with "exit status: 101": `PYO3_ENVIRONMENT_SIGNATURE="cpython-3.10-64bit" PYO3_PYTHON="/root/.local/pipx/venvs
/variantplaner/bin/python" PYTHON_SYS_EXECUTABLE="/root/.local/pipx/venvs/variantplaner/bin/python" "cargo" "rustc" "--message-format" "json-render-di
agnostics" "--manifest-path" "/tmp/pip-install-j_xk5xq3/variantplaner_4fe4e62d6d1842cfa9955433921c26d3/variantplaner_rs/Cargo.toml" "--release" "--lib
"`
      Error: command ['maturin', 'pep517', 'build-wheel', '-i', '/root/.local/pipx/venvs/variantplaner/bin/python', '--compatibility', 'off'] returned
 non-zero exit status 1
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for variantplaner-rs
ERROR: Could not build wheels for variantplaner-rs, which is required to install pyproject.toml-based projects

```

**Comment by natir** [comment]
I think your version of rust is too 'old' with rust 1.73 I have no problem. In fact, the `ilog` functions are only available from version 1.67, so I can't guarantee from which rust version the polars dependencies can be installed without problems.

**Comment by antonylebechec** [comment]
Got it!
I used:
```curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh```
